lease/lessor.go

```diff
@@ -85,7 +100,10 @@ func (le *lessor) Revoke(id uint64) error {
 
 	delete(le.leaseMap, l.id)
 
-	// TODO: remove attached items
+	for item := range l.itemSet {
+		le.KV.DeleteRange([]byte(item.key), []byte(item.endRange))
```

Comment:
I don't understand this. The range [key, endRange] may cover the keys that don't want to be removed:
1. Another users adds a new key in this range when the lease is going on
2. the key is used in another lease

Maybe I don't understand lease mechanism yet. Could you explain more?